### PR TITLE
Commerce 7507

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
Adding a new commit for this PR https://github.com/liferay-commerce/liferay-portal/pull/1845/

@brianikim

### CODE REVIEW

**Comment by dsitu** [comment]
@andrea-ale-sbarra the baseline commit is causing conflicts

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -268,6 +268,8 @@ action.VIEW_COMMERCE_NOTIFICATION_QUEUE_ENTRIES=View Notification Queue Entries
 action.VIEW_COMMERCE_NOTIFICATION_TEMPLATES=View Notification Templates
 action.VIEW_COMMERCE_ORDER_TYPES=View Commerce Order Types
 action.VIEW_COMMERCE_ORDERS=View Orders
+action.VIEW_COMMERCE_PRODUCT_ATTACHMENTS=View Attachments
+action.VIEW_COMMERCE_PRODUCT_IMAGES=VIew Images
```
Comment (dsitu) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: @andrea-ale-sbarra typo here
```suggestion
action.VIEW_COMMERCE_PRODUCT_IMAGES=View Images
```

**Pull request closed without merging**